### PR TITLE
Update model primary key for dynamic retrieval

### PR DESCRIPTION
**Pull request by DavidSche**
删除、更新、查询操作支持非id名称的主键，主键名称从sqlalchemy 模型定义中获取

### CODE REVIEW

**Comment by wu-clan** [comment]
令人兴奋的，我会稍后 review 和 merge。🙏🙏🙏

File: sqlalchemy_crud_plus/crud.py [hunk 1]
```diff
@@ -13,6 +13,18 @@
...
+    def _get_primary_key(self):
+        """
+        Dynamically retrieve the primary key column(s) for the model.
+        """
+        mapper = inspect(self.model)
+        primary_key = mapper.primary_key
+        if len(primary_key) == 1:
+            return primary_key[0]
+        else:
+            raise ValueError("Composite primary keys are not supported")
```
Comment (wu-clan) [inline sqlalchemy_crud_plus/crud.py:27]: A custom error class is needed here.

**Review by wu-clan — APPROVED** [review]

File: sqlalchemy_crud_plus/crud.py (hunk 1 above)
Comment (wu-clan) [inline sqlalchemy_crud_plus/crud.py:27]: A custom error class is needed here.

**Review by wu-clan — CHANGES_REQUESTED** [review]

**Comment by wu-clan** [comment]
Can you update the documentation code for this?

Also, the ruff lintting and formatting needs to be executed

**Comment by wu-clan** [comment]
I can push to this PR if needed.

File: sqlalchemy_crud_plus/crud.py (hunk 1 above)
Comment (DavidSche) [reply sqlalchemy_crud_plus/crud.py]: I have update the code ,please review ,thanks

**Comment by wu-clan** [comment]
@DavidSche 这太酷了

非常感谢

**Review by wu-clan — APPROVED** [review]

**Pull request merged**